firmware/user/modes/mode_ddr.c

```diff
@@ -231,39 +263,52 @@ void ICACHE_FLASH_ATTR ddrEnterMode(void)
 
     timerDisarm(&ddr->TimerHandleLeds);
     timerSetFn(&ddr->TimerHandleLeds, ddrLedFunc, NULL);
-    timerArm(&ddr->TimerHandleLeds, LEDS_TIMER, true);
 
     timerDisarm(&ddr->TimerAnimateSuccessMeter);
     timerSetFn(&ddr->TimerAnimateSuccessMeter, ddrAnimateSuccessMeter, NULL);
     timerArm(&ddr->TimerAnimateSuccessMeter, 40, true);
 
     timerDisarm(&ddr->TimerSongDuration);
     timerSetFn(&ddr->TimerSongDuration, ddrSongDurationFunc, NULL);
-    timerArm(&ddr->TimerSongDuration, SONG_DURATION, true);
+
+    timerDisarm(&ddr->TimerAdvanceHighScores);
+    timerSetFn(&ddr->TimerAdvanceHighScores, ddrAdvanceHighScores, NULL);
 }
 
 static void ICACHE_FLASH_ATTR ddrMenuCb(const char* menuItem)
 {
     if(ddr_easy == menuItem)
     {
-        ddrStartGame(80, 0.8f, 0);
+        ddrStartGame(80, 0.8f, 0, DDR_EASY);
     }
     else if (ddr_medium == menuItem)
     {
-        ddrStartGame(85, 2.f, 10);
+        ddrStartGame(85, 2.f, 10, DDR_MEDIUM);
     }
     else if (ddr_hard == menuItem)
     {
-        ddrStartGame(90, 3.f, 40);
+        ddrStartGame(90, 3.f, 40, DDR_HARD);
     }
-
     else if (ddr_quit == menuItem)
     {
         switchToSwadgeMode(0);
     }
+    else if (ddr_highscores == menuItem)
+    {
+        ddrGoToScores();
+    }
 }
 
-static void ICACHE_FLASH_ATTR ddrStartGame(int tempo, float eighthNoteProbabilityModifier, int restAvoidanceProbability)
+static void ICACHE_FLASH_ATTR ddrGoToScores()
+{
+    ddr->mode = DDR_HIGHSCORE;
+    ddr->isNewHighScore = false;
+    ddr->currentDifficultyType = DDR_HARD;
+    timerDisarm(&ddr->TimerAdvanceHighScores);
+    timerArm(&ddr->TimerAdvanceHighScores, HIGH_SCORE_PERIOD, true);
```

Comment:
Oh, I get it now. I flashed your changes before reviewing the code and was mashing the buttons on the high score screen to get it to show easy and medium. Having buttons cycle the pages here is more intuitive than doing it on a timer. You can put "<" and ">" in the bottom left and right corners to indicate cycle-ability, like on the root menu screen